src/appsignal/agent.py

```diff
@@ -16,6 +16,15 @@ class Agent:
 
     def start(self, config: Config) -> None:
         config.set_private_environ()
+
+        if self.architecture_and_platform() == ["any"]:
+            print(
+                "AppSignal agent is not available for this platform. ",
+                "The integration is now running in no-op mode therefore ",
+                "no data will be sent to AppSignal.",
```

Comment:
```suggestion
                "AppSignal agent is not available for this platform. "
                "The integration is now running in no-op mode therefore "
                "no data will be sent to AppSignal."
```

(the commas cause it to be sent as three different arguments to `print()` -- so there's a double space after each break: the space in the string, and the space `print()` puts between arguments)